### PR TITLE
remark for mobile implemented

### PR DESCRIPTION
**Pull request by evidentti**

### CODE REVIEW

File: app/loc/lang-fi.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "STR_OTHER_MATERIAL": "Muu materiaali",
     "STR_ADDITIONAL_MATERIAL": "Oheismateriaali",
     "STR_DECISION_HISTORY": "Päätöshistoria",
-    "STR_OWN_REMARK": "Oma merkintä",
+    "STR_REMARK": "Oma merkintä",
```
Comment (tkenakka) [inline app/loc/lang-fi.json:75]: There is already a "STR_OWN_REMARK"

File: app/scripts/controllers/remark.ctrl.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* # remarkCtrl
+* Controller of the dashboard
+*/
+angular.module('dashboard')
+    .controller('remarkCtrl', ['$log', '$scope', 'StorageSrv', 'CONST', function ($log, $scope, StorageSrv, CONST) {
+        $log.debug("remarkCtrl: CONTROLLER");
+        var self = this;
+        self.topic = StorageSrv.getKey(CONST.KEY.TOPIC);
+
+        if (!angular.isObject(self.topic)) {
```
Comment (tkenakka) [inline app/scripts/controllers/remark.ctrl.js:20]: Should this check be before assigning, and skip if it's not valid? Then template can assume property is etiher a `{}` or a valid object? Just a thought.

**Comment by tkenakka** [comment]
LGTM but with comments + presuming merge conflict is resolved.

File: app/loc/lang-fi.json (hunk 1 above)
Comment (evidentti) [inline app/loc/lang-fi.json:75]: Yes it is and it is renamed to STR_REMARK.

**Pull request merged**